widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt

```diff
@@ -17,7 +17,7 @@ internal class CallNotificationUseCase(
             video = getVideoDirection(visitorMedia, operatorMedia)
         } catch (error: IllegalStateException) {
             // One of the impossible scenario was detected during parsing of the states which is not supported by current GliaHub
-            Logger.e(TAG, "Unsupported request to show/update/hide media call notification", error)
+            Logger.i(TAG, "Unsupported request to show/update/hide media call notification \nDetails: ${error.message}")
```

Comment:
This probably will cause merge conflicts with 'development' branch later since it is already implemented there.

Reply:
Most probably, indeed. I also thought about it. Okay, rolled back this change. Thank you.